src/support/utils.h

```diff
@@ -29,75 +29,31 @@ int cmp_lt(void *a, numerictype_t atag, void *b, numerictype_t btag);
 int cmp_eq(void *a, numerictype_t atag, void *b, numerictype_t btag,
            int equalnans);
 
-#ifdef __x86_64__
-#  define LEGACY_REGS "=Q"
+#if defined(__clang__) || (defined(__GNUC__) && (__GNUC__ > 4 || __GNUC_MINOR__ >= 8))
+#define bswap_16(x) __builtin_bswap16(x)
```

Comment:
Apparently gcc 4.7 is mostly-in-theory the minimum for llvm since it wasn't the default system compiler on many particularly commonly tested distros.